ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionlayer/ExecutionLayerChannel.java

```diff
@@ -68,6 +70,12 @@ public SafeFuture<PayloadStatus> engineNewPayload(
           return SafeFuture.completedFuture(PayloadStatus.SYNCING);
         }
 
+        @Override
+        public SafeFuture<List<ClientVersion>> engineGetClientVersion(
+            final ClientVersion clientVersion) {
+          return SafeFuture.completedFuture(null);
```

Comment:
not sure i love returning completed of null for a list...

Reply:
Makes sense. Empty list is better.